src/abi/composableStableFactory.V2.ts

```diff
@@ -1,4 +1,4 @@
-export const composableStableFactoryV5Abi = [
+export const composableStableFactoryV2Abi = [
```

Comment:
I think it's still nice to mention the factory version as well, since this might change in the future.
Here are a few options for us to evaluate:
- `composableStableFactoryV5Abi_V2`
- `composableStableFactoryV5Abi_balancerV2`
- `composableStableFactoryV5Abi_vaultV2`